zipline-gradle-plugin/README.md

```diff
@@ -8,9 +8,12 @@ The plugin compiles JavaScript files into `.zipline` files. These binary files a
 
 The plugin requires setting the following variables which are used in compilation.
 
-- `mainModuleId`: This is the JS module that your application's entrypoint function is in. This will usually be the last key in the `modules` map in the generated ZiplineManifest which is sorted topologically.
 - `mainFunction`: This is the fully qualified function call to start your application, note it does include the trailing `()` to initiate the call.
```

Comment:
I missed this in my last PR. . . . change the word `requires`  to `offers` on line 9 ?

Reply:
Oh drat my PR isn’t merged. Rebase on that??